test/upgrade/mzworkflows.py

```diff
@@ -28,13 +29,18 @@
 # Determine the list of versions to be tested
 #
 
-min_tested_tag = VersionInfo.parse("0.8.0")
-all_tags = get_version_tags(fetch=False)
-all_tested_tags = []
+min_tested_tag = VersionInfo.parse(os.getenv("MIN_TESTED_TAG", "0.8.0"))
```

Comment:
I'd like to soon allow workflows to accept command-line arguments, but this seems fine for now!